types/stream-to-blob/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "ES6"
+        ],
+        "esModuleInterop": true,
```

Comment:
Enabling `"esModuleInterop"` is non-standard on this repository, and should be avoided when possible.
Could you remove this option from this config?
I think if you also follow my recommendation on the import syntax used on the test, then there shouldn't be an issue.

Reply:
@gabritto thank for review, I fixed it as per your request.